neuralmonkey/encoders/sentence_encoder.py

```diff
@@ -117,6 +117,10 @@ def _attention_mask(self):
         return self.input_mask
 
     @property
+    def states_mask(self):
+        return self.input_mask
+
```

Comment:
Jestli ne, dal bych enkodérům (respektive všem `ModelPart` s nějakým sekvenčním vstupem) `input_mask` a všechno maskoval podle toho. `attention_mask` je použitá na pár místech a přejmenovat to je jednoduchý.